### ISSUE TITLE
Parameter names in stubs are inconsistent with official docs leading to false InvalidNamedArgument errors

### ISSUE THREAD

**Issue by tminich** [vimeo/psalm#6803]
A lot of the parameter names in the stubs psalm uses do not match the official doc. If you use named arguments in PHP 8 with vanilla PHP functions you get a lot of false InvalidNamedArgument errors.

Example:
```php
// Official
function array_key_exists(string|int $key, array $array): bool {}

// Stub
function array_key_exists($key, array $search) : bool {}
```

**Comment by tminich** [comment]
https://psalm.dev/r/33f4798e90
